reboot_toolkit/toolkit_backend.py

```diff
@@ -399,6 +403,168 @@ def load_games_to_df_from_s3_paths(
 
     return all_games_df
 
+def load_sampled_games_to_df_from_s3_paths(

```

Comment:
Is this mostly copied and pasted from the other function? Can we unify this function with the other one so that we don't need to maintain both functions in parallel?

Reply:
Done.